src/interfaces.ts

```diff
@@ -70,12 +70,19 @@ export enum PluginType {
   "AI" = "AI",
 }
 
-export interface IPlugin {
+interface IPlugin {
   username?: string;
   type: PluginType;
   name: string;
   url: string;
   products: Product;
   enabled: boolean;
+}
+
+export interface IPluginIn extends IPlugin {
+  collection_uids: { uid: string; is_invite_pending: "True" | "False" }[];
```

Comment:
What's with the string literals here, shouldn't it just be a boolean?

Reply:
oh, yes! I wasn't able to get the API to return a `boolean`, so that's a dirty little fix.. now properly fixed by adding a schema for the data passed in the request. thanks!